overlord/ifacestate/helpers.go

```diff
@@ -197,12 +198,14 @@ func (m *InterfaceManager) regenerateAllSecurityProfiles() error {
 				continue // Test backends have no name, skip them to simplify testing.
 			}
 			// Refresh security of this snap and backend
-			if err := backend.Setup(snapInfo, opts, m.repo); err != nil {
-				// Let's log this but carry on without writing the system key.
-				logger.Noticef("cannot regenerate %s profile for snap %q: %s",
-					backend.Name(), snapName, err)
-				shouldWriteSystemKey = false
-			}
+			timings.Run(tm, "setup-security-backend", fmt.Sprintf("setup security backend %q for snap %q", backend.Name(), snapInfo.InstanceName()), func(nesttm timings.Measurer) {
```

Comment:
Can we test in ifacemgr_test.go (or somewhere) that the timings are actually run? Or does that not make much sense?

Reply:
Added.